src/Addons/Provider/LinksProvider.php

```diff
@@ -161,7 +161,7 @@ public function getSignUpLink(): string
     {
         $isoCode = mb_strtolower($this->getIsoCode());
 
-        return $this->translator->trans('https://accounts.distribution.prestashop.net/fr/sign-up', [], 'Modules.Mbo.Links')
+        return $this->translator->trans('https://accounts.distribution.prestashop.net/' . $isoCode . '/sign-up', [], 'Modules.Mbo.Links')
```

Comment:
Why not just put `en` as it's a translatable string ?
So we will be able to manage more easily the existing language or not, and always have the default to `en`?

Reply:
Ok. Fine for me